docs/pages/includes/tls-certificate-setup.mdx

```diff
@@ -24,6 +24,11 @@ deployments, use your own private key and certificate.
     ```
 
     Port 443 on your Teleport Proxy Service host must allow traffic from all sources.
+
+     <Admonition type="notice" title="Acme Certificates only for single Proxy" >
+       The Acme Let's Encrypt option is only a single proxy instance and is not supported for highly available configurations.
```

Comment:
Let's Encrypt is just one of many different ACME-compatible services, should we be calling it out specifically?

Also, ACME is an acronym and should be capitalized.